bin/fillTestData/Users.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
 
-var _ = require('lodash'),
-    path = require('path'),
-    mongooseService = require(path.resolve('./config/lib/mongoose')),
-    chalk = require('chalk'),
-    yargs = require('yargs'),
-    faker = require('faker'),
-    fs = require('fs'),
-    moment = require('moment'),
-    mongoose = require('mongoose'),
-    async = require('async'),
-    config = require(path.resolve('./config/config')),
-    cities = JSON.parse(fs.readFileSync(path.resolve('./bin/fillTestData/data/Cities.json'), 'utf8')),
-    savedUsers = 0,
+const _ = require('lodash'),
+      path = require('path'),
+      mongooseService = require(path.resolve('./config/lib/mongoose')),
+      chalk = require('chalk'),
+      yargs = require('yargs'),
+      faker = require('faker'),
+      fs = require('fs'),
+      moment = require('moment'),
+      mongoose = require('mongoose'),
+      config = require(path.resolve('./config/config')),
+      cities = JSON.parse(fs.readFileSync(path.resolve('./bin/fillTestData/data/Cities.json'), 'utf8'));
```

Comment:
Unless the cities get edited + used elsewhere, `cities = path.resolve('./bin/fillTestData/data/Cities')` [may be sufficient](https://nodejs.org/dist/latest-v11.x/docs/api/modules.html#modules_file_modules) here.

Reply:
@mrkvon - Thanks! I am assuming you meant `cities = require(path.resolve('./bin/fillTestData/data/Cities.json'));`. Just added the change.